resources/core/charts/kubeless/templates/tests/test-kubeless-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-kubeless-config
+  namespace: kyma-system
```

Comment:
`namespace: kyma-system` is redundant, lets leave it to helm as we do in other resources used for testing

Reply:
Done.